### PR TITLE
Fixes #886 Padding removed to remove text overflow

### LINKED ISSUE #886: Texts overflowing to the left in profile.html
## Description
In profile.html , the name is not completely visible since a part of it is overflowing to the left.

## Mocks
screenshot from 2018-12-10 18-55-21

### Update [Required]
- [X] Identify css element causing the problem and removing the error.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by Tlazypanda**
# Description
Padding is removed which removes the text overflow to the left in profile.html.The name is now completely visible as shown in the mock.

# MOCKS
image

Fixes #886 

# Type of Change:
- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

# Checklist:
**Delete irrelevant options.**
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Monal5031** [comment]
@Tlazypanda travis.

**Comment by Tlazypanda** [comment]
@01shobitha Yeah still working :sweat_smile:  @Monal5031 Travis build passed :smile:

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Installation
 The VMS project uses the [Django](https://www.djangoproject.com/) web
 framework and [Python](https://www.python.org/).
 
-To get started, read the [Installation Guide](https://github.com/systers/vms/blob/develop/docs/Installation_Guide.md).
+To get started, read the [Installation Guide](https://github.com/systers/vms/blob/develop/aut_docs/Installation_Setup.md).
```
Comment (Monal5031) [inline README.md]: Unrelated change.

File: requirements.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ coveralls == 0.5
 Django == 1.11.0
 docopt == 0.6.2
 flake8 == 3.4.1
-psycopg2 == 2.7.3
+psycopg2 == 2.7.4
```
Comment (Monal5031) [inline requirements.txt]: Unrelated change.

**Comment by aayush-05** [comment]
@Monal5031 shall I open a new pr for this ? This one got one year old and I don't think it it being worked on now.

**Comment by Monal5031** [comment]
Closing due to inactivity
